plutus-core/plutus-core/src/PlutusCore/Builtin/KnownKind.hs

```diff
@@ -23,7 +25,7 @@ class KnownKind k where
     knownKind :: SingKind k
 
 -- | Plutus only supports lifted types, hence the equality constraint.
-instance rep ~ 'LiftedRep => KnownKind (TYPE rep) where
+instance rep ~ LiftedRep => KnownKind (TYPE rep) where
```

Comment:
Just spent several minutes figuring out why we need to turn off the warning and why the tick is no longer OK. Please do document subtle changes like that.

I'll sneak some documentation into one of the future PRs.